samples/apps/copilot-chat-app/webapp/tests/chat.test.ts

```diff
@@ -0,0 +1,43 @@
+﻿import { expect, test } from '@playwright/test';
+
+test('get response from bot', async ({ page }) => {
+    // Make sure the server is running.
+    await page.goto('https://localhost:40443/healthz');
+    expect(page.getByText('Healthy')).toBeDefined();
+
+    await page.goto('/');
+    // Expect the page to contain a "Login" button.
+    await page.getByRole('button').click();
+    // Clicking the login button should redirect to the login page.
+    await expect(page).toHaveURL(new RegExp('^' + process.env.REACT_APP_AAD_AUTHORITY));
+    // Login with the test user.
+    await page.getByPlaceholder('Email, phone, or Skype').click();
+    await page.getByPlaceholder('Email, phone, or Skype').fill(process.env.REACT_APP_TEST_USER_ACCOUNT as string);
+    await page.getByRole('button', { name: 'Next' }).click();
+    await page.getByPlaceholder('Password').click();
+    await page.getByPlaceholder('Password').fill(process.env.REACT_APP_TEST_USER_PASSWORD as string);
+    await page.getByRole('button', { name: 'Sign in' }).click();
+
+    // Select No if asked to stay signed in.
+    const isAskingStaySignedIn = await page.$$("text='Stay signed in?'");
+    if (isAskingStaySignedIn) {
+        await page.getByRole('button', { name: 'No' }).click();
+    }
+
+    // After login, the page should redirect back to the app.
+    await expect(page).toHaveTitle('Copilot Chat');
+
+    // Send a message to the bot and wait for the response.
+    const responsePromise = page.waitForResponse('**/chat');
+    await page.locator('#chat-input').click();
+    await page.locator('#chat-input').fill('Hi!');
```

Comment:
I feel like this would put us into a place where we are testing the behavior of the models not the front end & backend interactions. 

Maybe we can create specific tests for multi-turn interactions that will make sure the bot responds as expected.